components/Footer/Footer.jsx

```diff
@@ -1,5 +1,87 @@
-const Footer = (props) => {
-    const { children } = props
-    return <footer>{children}</footer>
+import Image from 'next/image'
+import britoFlag from '../../public/images/brito-logo-1.svg'
+import britoWordMark from '../../public/images/brito-logo-2.svg'
+import styles from '@/styles/Footer.module.css'
+import labels from '@/constants/labels.en'
+import { Icon } from '@/components'
+
+const Footer = () => {
+    return (
+        <div>
+            <div className={`${styles.footerContainer} flex-row-space-between`}>
+                <div className={styles.logoAndSocial}>
+                    <div className={styles.logoAndSocial}>
+                        <Image alt='brito flag' src={britoFlag} className={styles.logo} />
+                        <Image alt='brito wordMark' src={britoWordMark} className={styles.logo} />
```

Comment:
It's broken on mobile

<img width="764" alt="Screenshot 2023-03-08 at 10 22 04 PM" src="https://user-images.githubusercontent.com/11657599/223908479-054cf8d4-1e37-4ad5-bd58-e5cb9867619b.png">

Reply:
The fonts on the mobile mockup are different from the desktop version, and those are not in the globals
I couldn't fix the spacing in the flex between the logo and social media icon, i need help with this

Reply:
Suggestions:
1. _The fonts on the mobile mockup are different from the desktop version_
1.1 Use the same font for both and ask them about that (add a comment in https://github.com/brito-ca/brito-website/issues/36 and put it in the WhatsApp channel)
1.2 The font now comes from the @next/font and is in the AppLayout component, but its use should be the same as before. Let me know of any specific issues.
2. _I couldn't fix the spacing in the flex between the logo and social media icon, I need help with this_
2.1 I can take a look at that. Is it to what? Is it the same as in the mockup?

Reply:
1. Ok, I will add a comment.
2. Yes is the same as the mockup
              When the width is around 700 px the flex is wrapping like the screenshot Marcelo sent before.
              When the width is smaller than 600px the spacing between the logos is strange, looks like they have a margin/padding that I could not fix
![Screen Shot 2023-03-15 at 19 20 04](https://user-images.githubusercontent.com/110069912/225465458-a8b695a1-803d-4803-8778-4d0a6581ead1.png)

Reply:
I sent a PR to yours (https://github.com/brito-ca/brito-website/pull/47). I took the liberty to change things from the mockup to make it easy.

Things that don't make sense:
- Company --> changed to BRITO
- Removed the Network list as discussed in the meeting
- Rearranged the logo content and the "nav" on top of the other for mobile
- Rearranged the bottom footer to a better disposition on mobile
- Refactored the props passed
- refactored the styling (check if you can grasp my changes. If not, we can connect. So I can explain it)
- Removed the fab when on the bottom of the page to avoid being in front of the text